src/components/Editor/Navigator.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+const React = require("react");
+const { DOM: dom, PropTypes } = React;
+const Svg = require("../shared/Svg");
+
+require("./Navigator.css");
+
+type NavigatorButtonsType = {
+  handleNext: any,
+  handlePrev: any,
+  nextClass?: string,
+  prevClass?: string,
+  nextTooltip?: string,
+  prevTooltip?: string
+};
+
+function NavigatorButtons({ handleNext, handlePrev, nextDisabled, prevDisabled, nextClass, prevClass, nextTooltip, prevTooltip }: NavigatorButtonsType) {
+  return dom.div({
+    className: "nav-buttons"
+  },
+    Svg("arrow-down", {
+      className: "nav "+(nextClass ? `nav-btn-${nextClass}` : "next-btn"),
```

Comment:
I think you could use the `classnames` utility and probably avoid having props for the class names. They could be reused.